pinot-common/src/main/java/org/apache/pinot/common/utils/primitive/ByteArray.java

```diff
@@ -45,19 +45,13 @@ public int length() {
     return _bytes.length;
   }
 
-  /**
-   * Static utility function to convert a byte[] to Hex string.
-   *
-   * @param bytes byte[] to convert
-   * @return Equivalent Hex String.
-   */
-  public static String toHexString(byte[] bytes) {
-    return Hex.encodeHexString(bytes);
+  public String toHexString() {
+    return BytesUtils.toHexString(_bytes);
```

Comment:
why did we move this to BytesUtils class?

Reply:
BytesUtils is the util class for all byte[] related methods. ByteArray is the comparable wrapper for byte[], which should not be used as util class.